languages/perl5/README.md

```diff
@@ -12,7 +12,7 @@ This area will contain everything needed to launch the Perl 5 track, including:
 
 Before we publicize requesting contribution for this language, the following steps should be done.
 
-- [ ] Have a kick-off discussion between track maintainers
+- [x] Have a kick-off discussion between track maintainers
 - [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
```

Comment:
Should this link be fixed to reflect the actual file needed to edit?

Reply:
What link, @kotp ? Sorry, I didn't understand. Do you say the _maintainers.md_ file?

Reply:
`[C#](../csharp/maintainers.md)` is the linked file... should we link it to the actual file that should be filled out, per the check box?

Reply:
This should be fixed, possibly in a separate PR.